applications/app/controllers/MediaAtomEmbedController.scala

```diff
@@ -0,0 +1,66 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.ItemResponse
+import com.gu.contentatom.thrift.{Atom => AtomApi}
+import common._
+import model.Cached.{RevalidatableResult, WithoutRevalidationResult}
+import model._
+import play.api.mvc._
+
+import scala.concurrent.Future
+import contentapi.ContentApiClient
+import model.content.MediaAtom
+
+class MediaAtomEmbedController(contentApiClient: ContentApiClient) extends Controller with Logging with ExecutionContexts {
+
+  def render(id: String) = Action.async { implicit request =>
+    lookup(s"atom/media/$id") map {
+      case Left(model) => renderMediaAtom(model)
+      case Right(other) => renderOther(other)
+    }
+  }
+
+  def make(apiAtom: Option[AtomApi]): Option[MediaAtom] = {
+    apiAtom map {
+      MediaAtom.make
+    }
+  }
```

Comment:
you can probably make this one line `def make(apiAtom: Option[AtomApi]): Option[MediaAtom] = apiAtom.map(MediaAtom.make)`